src/store/modules/aeternity.js

```diff
@@ -114,11 +114,10 @@ export default {
           commit('setSdk', { instance });
           await dispatch('initTippingContractIfNeeded');
         }
-      } catch (err) {
-        commit('setBackendStatus', false, { root: true });
-        return;
+        return true;
+      } catch (error) {
+        return error;
```

Comment:
who will handle the returned (not thrown) error? proposing to just remove `try`/`catch`

Reply:
right, I added that because of lint. will remove